pkg/kubelet/apis/config/helpers_test.go

```diff
@@ -226,5 +226,6 @@ var (
 		"TypeMeta.Kind",
 		"VolumeStatsAggPeriod.Duration",
 		"VolumePluginDir",
+		"RootDirectory",
```

Comment:
this should be in the kubeletConfigurationPathFieldPaths list, *not* the kubeletConfigurationNonPathFieldPaths list